satellite/audit/service.go

```diff
@@ -32,7 +31,8 @@ type Config struct {
 	MinDownloadTimeout time.Duration `help:"the minimum duration for downloading a share from storage nodes before timing out" default:"25s"`
 	MaxReverifyCount   int           `help:"limit above which we consider an audit is failed" default:"3"`
 
-	Slots int `help:"number of reservoir slots allotted for nodes, currently capped at 2" default:"1"`
+	Slots         int `help:"number of reservoir slots allotted for nodes, currently capped at 2" default:"1"`
+	NodesToSelect int `help:"number of nodes' reservoirs to randomly select segments from" default:"100"`
```

Comment:
I'd call this `NumNodesToSelect` or `NodeSelectCount` or something like that. The name `NodesToSelect` does not imply that it is a number to me.